library/src/main/java/pl/netigen/rewards/LayoutMargins.java

```diff
@@ -0,0 +1,66 @@
+package pl.netigen.rewards;
+
+import android.view.View;
+
+import androidx.constraintlayout.widget.ConstraintLayout;
+
+public class LayoutMargins {
+
+    private Integer left;
+    private Integer top;
+    private Integer right;
+    private Integer bottom;
+
+    private LayoutMargins() {
+    }
+
+    void setNewLayoutParamsForView(View view){
+        ConstraintLayout.LayoutParams layoutParams = (ConstraintLayout.LayoutParams) view.getLayoutParams();
```

Comment:
ConstraintLayout maybe can be more general ViewGroup.MarginLayoutParams

Reply:
Done